Dockerfile

```diff
@@ -2,7 +2,10 @@
 FROM python:3.10-slim-buster
 
 WORKDIR /app
-COPY . /app/src
+COPY requirements.txt pyproject.toml setup.cfg setup.py /app/src/
```

Comment:
This works but I think it'd be cleaner and more expensive to use the docker ignore file to exclude what we don't want, rather than making that implicit by adding everything we do want